examples/versatileab/reference/registers-armv7r-none-eabi.out

```diff
@@ -1,5 +1,8 @@
 MIDR { implementer=0x41 variant=0x1 arch=0xf part_no=0xc15 rev=0x3 }
 CPSR { N=0 Z=1 C=1 V=0 Q=0 J=0 E=0 A=1 I=1 F=1 T=0 MODE=Ok(Sys) }
+Mpidr(3221225472)
+SCTLR { IE=0 TE=0 NMFI=0 EE=0 U=0 FI=0 DZ=0 BR=0 RR=0 V=0 I=0 Z=0 SW=0 C=0 A=0 M=0 } before setting C, I and Z
```

Comment:
BR became 0 here for some reason

Reply:
It's fine. It's just the start up state of the processor.